docs: updated sample app readme (#25)
dequelabs/axe-devtools-ios-sample-app: README.md

### For Manual Testing

> 1. To manually test this app, use the [Mobile Analyzer](https://github.com/dequelabs/axe-devtools-mobile-analyzer).

Reviewer: Do we consider using the analyzer manual testing? Or just another automated testing scenario with a different product?

Author: That's covered under step 6 in the Analyzer setup guide.

Reviewer: I wasn't saying that it wasn't considered manual testing, just confirming because in my head I think of something different when I hear 'manual testing' (I think of our old setup where you had to tap the FAB), so I just wanted to confirm, that's all. But given what Kim said about this being 'manual from customer's perspective'  it makes sense to me.

Author: Sounds good

> 1. To manually test this app, use the [Mobile Analyzer](https://github.com/dequelabs/axe-devtools-mobile-analyzer).

Reviewer 2: ```suggestion
1. To manually test this, or other apps, use [Deque's Mobile Analyzer](https://docs.deque.com/devtools-mobile-analyzer).
```

Author: Sounds good

### For Automated Testing

1. Add your Deque API Key to `Login.swift`.

> 2. Once you have a device or simulator ready to go, open `SampleUITests-XCUI.swift`, and tap on the diamond to run this suite. The simulator will launch, and the tests will run.

Reviewer 2: I think the file needs to be renamed in this branch as well. Could also remove `-XCUI` all together since there is only one version now.

Reviewer 2: @atristannewman  ^ still needs to be addressed. 
can either remove the rename, or add the renamed file to this PR

Author: Good call
